contracts/core/OriginCore.sol

```diff
@@ -478,9 +508,38 @@ contract OriginCore is OriginCoreInterface, OriginCoreConfig {
             _initialTransactionRoot
         );
 
-        reportedHeaders[kernelHash] = MetaBlock.Header(genesisKernel, genesisTransition);
+        bytes32 transitionHash = MetaBlock.hashAuxiliaryTransition(
+            auxiliaryCoreIdentifier,
+            kernelHash,
+            0,
+            bytes32(0),
+            _initialAuxiliaryGas,
+            0,
+            bytes32(0),
+            _initialTransactionRoot
+        );
+
+        metaBlockHash_  = MetaBlock.hashMetaBlock(
+            kernelHash,
+            transitionHash
+        );
+
+        reportedHeaders[metaBlockHash_] = MetaBlock.Header(genesisKernel, genesisTransition);
 
-        return kernelHash;
+        /* Open Kernel for height 1. */
+        openKernel = MetaBlock.Kernel(
+            1,
+            metaBlockHash_,
+            initialValidators,
+            validatorsWeights
```

Comment:
The change in validators should be an empty array for meta-block at height `1`. No validators can enter or leave during meta-block `0`.

Reply:
It's empty array only, I have changed the name if it's confusing.